dev-tools/mage/settings.go

```diff
@@ -143,11 +149,9 @@ func init() {
 		panic(fmt.Errorf("failed to parse FIPS env value: %w", err))
 	}
 
-	if FIPSBuild {
-		err := yaml.Unmarshal(fipsConfigRaw, &FIPSConfig)
-		if err != nil {
-			panic(fmt.Errorf("failed to parse FIPS config: %w", err))
-		}
+	err = yaml.Unmarshal(fipsConfigRaw, &FIPSConfig)
+	if err != nil {
+		panic(fmt.Errorf("failed to parse FIPS config: %w", err))
```

Comment:
Why did you remove the `if FIPSBuild { .. }` clause?

Reply:
It should be safe to read the FIPS settings in any case, having the initialization happen every time we can spot issues quicker.

I could add back the `if FIPSBuild` clause but for a few milliseconds extra on a one-time operation I preferred to parse the settings in any case